tutorials/how-to-roll-your-app-engine-managed-vms-app-back-to-a-previous-version-part-1.md

```diff
@@ -67,13 +66,11 @@ Create a file named `app.yaml` with the following contents:
 
 This configuration file tells App Engine how to run our app.
 
-Now create a file named `package.json with the following contents:
+Now create a file named `package.json` with the following contents:
 
-```json
-{
-    "name": "sample-app"
-}
-```
+    {
+        "name": "sample-app"
+    }
 
 Assuming you've created a project in the Google Cloud Platform Developers
 Console and installed the gcloud sdk locally, you should be ready to deploy.
```

Comment:
Generally, glcoud should be in code font although I think this should be changed to Cloud SDK?